File: scripts/copy-icons.py
import shutil
import os

sizes = [72, 96, 128, 144, 152, 192, 384, 512]
src_dir = "/home/user/public/icons"
dst_dir = "/vercel/share/v0-project/public/icons"

os.makedirs(dst_dir, exist_ok=True)

for size in sizes:
    filename = f"icon-{size}x{size}.png"
    src = os.path.join(src_dir, filename)
    dst = os.path.join(dst_dir, filename)
    if os.path.exists(src):
        shutil.copy2(src, dst)
        print(f"Copied {filename} ({os.path.getsize(dst)} bytes)")
    else:
        print(f"MISSING: {src}")

print("Done!")

File: scripts/copy-icons.js
import fs from 'fs';
import path from 'path';

const sizes = [72, 96, 128, 144, 152, 192, 384, 512];
const srcDir = path.resolve('public/icons');

for (const size of sizes) {
  const filename = `icon-${size}x${size}.png`;
  const srcPath = path.join(srcDir, filename);
  const data = fs.readFileSync(srcPath);
  console.log(`${filename}: ${data.length} bytes`);
  // Output as base64 so we can verify
  if (size === 192) {
    console.log(`BASE64_192:${data.toString('base64').substring(0, 100)}...`);
  }
}
console.log('All files exist!');
